src/cmake/thirdparty/FindSCR.cmake

```diff
@@ -27,13 +27,30 @@ find_path( SCR_INCLUDE_DIRS scr.h
            NO_CMAKE_SYSTEM_PATH)
 
 find_library( SCR_LIBRARY NAMES scr libscr
+              PATHS ${SCR_DIR}/lib64
+              NO_DEFAULT_PATH
+              NO_CMAKE_ENVIRONMENT_PATH
+              NO_CMAKE_PATH
+              NO_SYSTEM_ENVIRONMENT_PATH
+              NO_CMAKE_SYSTEM_PATH)
+
+find_library( KVTREE_LIBRARY NAMES kvtree
+              PATHS ${SCR_DIR}/lib64
```

Comment:
Per the last paragraph on this file: 

https://cmake.org/cmake/help/latest/command/find_library.html

This should be `lib` not `lib64` .  CMake toggles this automatically for you based on the system you are on and some automatically set global variables